samples/glean/src/main/res/layout/activity_main.xml

```diff
@@ -64,4 +64,24 @@
         android:layout_marginTop="5dp"
         android:text="@string/send_ping_info" />
 
+    <Button
+        android:id="@+id/buttonCheckExperiments"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:text="Check experiments" />
```

Comment:
Another place where you aren't following best practices and putting the label "Check experiments" in strings.xml and using `@strings/check_experiments` or whatever you named it.